### PR TITLE
Terraform plus ssm

### PR DESCRIPTION
**Pull request by redbearddog**
generate passwords
save passwords to ssm
get params from ssm
save endpoints to ssm
create webhook url

### CODE REVIEW

File: sandbox/euc101/main.tf [hunk 1]
```diff
@@ -1,41 +1,297 @@
...
+  common_tags = {
+    EnvClass    = var.env_class
+    Environment = local.env_name
+    Owner       = "DevOps"
+    Terraform   = "true"
+  }
+}
+
+########## Rassword Generation for RabbitMQ ##############
+resource "random_password" "mq_pass" {
```
Comment (anatolek) [inline sandbox/euc101/main.tf]: Please add a counter  with `rabbitmq_create`

File: sandbox/euc101/main.tf [hunk 2]
```diff
@@ -1,41 +1,297 @@
...
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+########## Rassword Generation for RDS ##############1
+resource "random_password" "rds_pass" {
```
Comment (anatolek) [inline sandbox/euc101/main.tf]: Please add a counter  with `rds_create`

File: sandbox/euc101/main.tf [hunk 3]
```diff
@@ -1,41 +1,297 @@
...
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+########## Get RabbitMQ User from SSM ##############
+data "aws_ssm_parameter" "mq_user" {
+  name = "/sandbox/euc101/mq_user"
```
Comment (anatolek) [inline sandbox/euc101/main.tf]: Please add a counter with `rabbitmq_create`.  
Also, change hard coded path with `env_class` and `env_name`.

File: sandbox/euc101/main.tf [hunk 4]
```diff
@@ -1,41 +1,297 @@
...
+}
+
+########## Get RDS User from SSM ##############
+data "aws_ssm_parameter" "rds_user" {
+  name = "/sandbox/euc101/rds_user"
```
Comment (anatolek) [inline sandbox/euc101/main.tf]: Please add a counter with `rds_create`.  
Also, change hard coded path with `env_class` and `env_name`.

File: sandbox/euc101/main.tf [hunk 5]
```diff
@@ -1,41 +1,297 @@
...
+  target_id        = module.ec2-instance-service["frontend"].id
+  port             = 5000
+}
+
+########## Get GIT_TOKEN from SSM ##############
+data "aws_ssm_parameter" "git_token" {
+  name = "git_token"
+}
+
+provider "github" {
```
Comment (anatolek) [inline sandbox/euc101/main.tf]: Move this provider to the beginning of the file

**Review by anatolek — CHANGES_REQUESTED** [review]

File: sandbox/euc101/main.tf (hunk 2 above)
Comment (redbearddog) [reply sandbox/euc101/main.tf]: done

**Pull request closed without merging**